hack/deploy_vault_config.sh

```diff
@@ -22,7 +22,8 @@ APP_CI_CA_CERT="$(oc --context=app.ci get configmap -n kube-public kube-root-ca.
 vault write auth/kubernetes/config \
     token_reviewer_jwt="${VAULT_KUBE_TOKEN}" \
     kubernetes_host=https://kubernetes.default.svc.cluster.local \
-    kubernetes_ca_cert="${APP_CI_CA_CERT}"
+    kubernetes_ca_cert="${APP_CI_CA_CERT}" \
+    issuer=https://ci-dv2np-oidc.s3.amazonaws.com
```

Comment:
For the sake of preserving knowledge, lets interpolate this please:
```suggestion
    issuer="$(oc get authentication.config cluster -o template="{{ .spec.serviceAccountIssuer }}")"
```